.distignore

```diff
@@ -1,10 +1,12 @@
 /.github
 /.git
 /.wordpress-org
-/node_modules
 /.sources
 /bin
 /docs
+/node_modules
+/packages
```

Comment:
This directory contains the packages we export from our plugin (`@woocommerce/blocks-checkout`, `@woocommerce/price-format` I think it needs to stay, for the same reasons as `assets` needs to stay.